.github/workflows/actions/verify-changesets/index.js

```diff
@@ -0,0 +1,139 @@
+import fs from 'node:fs/promises';
+
+// check if current file is the entry point
+if (import.meta.url.endsWith(process.argv[1])) {
```

Comment:
I use this check for testability. The code below is not executed when `.github/workflows/actions/verify-changesets/index.js` is imported in `.github/workflows/actions/verify-changesets/test.js`